3-WebApp-multi-APIs/Controllers/HomeController.cs

```diff
@@ -88,16 +86,24 @@ public async Task<IActionResult> Blob()
                     await blobClient.UploadAsync(stream);
                     message = "Blob successfully created";
                 }
-	        catch (MicrosoftIdentityWebChallengeUserException ex)
```

Comment:
I'm confused. What changed here? we never indent the catch ?

Reply:
Looks like your new catch missing one tab to be aligned with next catch.